src/foam/nanos/theme/Theme.js

```diff
@@ -752,6 +752,11 @@ foam.CLASS({
       class: 'Boolean',
       name: 'showNavSearch',
       value: true
+    },
+    {
+      class: 'String',
+      name: 'emailLinkRedirect',
```

Comment:
Can this go in the gateway token setup / service - as it's only applicable to iframes and the gateway token system?